.github/workflows/ci.yml

```
@@ -0,0 +1,92 @@
+          name: application-jar
+          path: build/libs/*.jar
+          retention-days: 7
+
+      - name: Publish test results
+        if: always()
+        uses: EnricoMi/publish-unit-test-result-action@v2
+        with:
+          files: |
+            build/test-results/**/*.xml
+
+
```

YAMLlint (1.38.0)

[error] too many blank lines (2 > 0)

(empty-lines)